test/babelish/test_android2csv.rb

```diff
@@ -50,4 +50,23 @@ def test_special_chars
     # clean up
     system("rm -rf ./" + csv_filename)
   end
+
+  def test_cdata_are_not_removed
+    csv_filename = "./test.csv"
+    filename = "test/data/android_cdata.xml"
+    headers = %w{variables german}
+
+    expected_output = [["html"], {filename => {"html" => "<![CDATA[<p>Text<p>]]>"}}]
+    converter = Babelish::Android2CSV.new(
+      :csv_filename => csv_filename,
+      :headers => headers,
+      :filenames => [filename])
+
+    output = converter.convert(false)
+
```

Comment:
I remove a line here which check if file exists.

Since the above `convert` has a `false`.  I think the file will be not exist?